app/assets/javascripts/init.js.coffee

```diff
@@ -13,4 +13,10 @@ App.init = ->
 # TODO may also need to do on pjax_modal change
 $ ->
   App.init()
-  $('.datepicker.enable-on-load').prop('disabled', false)
\ No newline at end of file
+  $('.datepicker.enable-on-load').prop('disabled', false)
+
```

Comment:
added this so that if you want to create a poller you just add a data attribute like `data-poll-every=30.seconds`